src/app/components/Billboard/BillboardCurationGrid/index.styles.ts

```diff
@@ -0,0 +1,134 @@
+import { css, Theme } from '@emotion/react';
+
+const styles = {
+  list: ({ isLite, mq }: Theme) =>
+    css({
+      listStyleType: 'none',
+      padding: 0,
+      margin: 0,
+      display: 'grid',
+      gap: '1rem',
+
+      [mq.GROUP_2_MAX_WIDTH]: {
+        gridTemplateColumns: '1fr',
+      },
+      [mq.GROUP_3_ONLY]: {
+        gridTemplateColumns: 'repeat(3, 1fr)',
+
+        '& > li:nth-of-type(4)': {
+          display: 'none',
+        },
+      },
+
+      [mq.GROUP_4_MIN_WIDTH]: {
+        gridTemplateColumns: 'repeat(4, 1fr)',
+      },
+
+      ...(isLite && {
+        li: {
+          '.promo-text': {
+            width: '100%',
+            paddingInlineStart: 0,
+          },
+        },
+      }),
+    }),
+
+  item: ({ spacings, mq, palette }: Theme) =>
+    css({
+      verticalAlign: 'top',
+      display: 'block',
+      width: '100%',
+
+      '.promo-image': {
```

Comment:
Since we're targeting CurationPromo class names and HTML elements, do we need to pass in the `billboardContext` stuff? Can we not just use the styling in here to apply the styles directly?

Reply:
Nice! Couple of type files can be reverted to remove the context boolean as well. Nice one 👍